extensions/extension-editing/src/extensionLinter.ts

```diff
@@ -33,6 +33,7 @@ const relativeBadgeUrlRequiresHttpsRepository = l10n.t("Relative badge URLs requ
 const apiProposalNotListed = l10n.t("This proposal cannot be used because for this extension the product defines a fixed set of API proposals. You can test your extension but before publishing you MUST reach out to the VS Code team.");
 const implicitActivationEvent = l10n.t("This activation event cannot be explicitly listed by your extension.");
 const redundantImplicitActivationEvent = l10n.t("This activation event can be removed as VS Code generates these automatically from your package.json contribution declarations.");
+const starActivation = l10n.t("Using '*' activation is usually a bad idea as it impacts performance.");
```

Comment:
Should we have a specific recommendation here, i.e. 'consider removing this in favor of a specific activation trigger'?